src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml

```diff
@@ -230,6 +233,16 @@ tests:
               type: int8u
               notValue: StartUpCurrentLevelValue
 
+    #This is a reset step that is needed to reset the value of the attribute to another value then the one used in 6b in order to ensure successful test repetitions.
```

Comment:
This is not great, since it introduces a divergence from the test plan in this certification test.

I know we have a step like that below already, but can we either get the test plan fixed to match the test or fix the way we run this test to match the test plan?